include/boost/test/impl/execution_monitor.ipp

```diff
@@ -534,23 +543,23 @@ system_signal_exception::report() const
         case BUS_ADRALN:
             report_error( execution_exception::system_fatal_error,
                           "memory access violation at address: 0x%08lx: invalid address alignment",
-                          m_sig_info->si_addr );
+                          (uintptr_t) m_sig_info->si_addr );
             break;
         case BUS_ADRERR:
             report_error( execution_exception::system_fatal_error,
                           "memory access violation at address: 0x%08lx: non-existent physical address",
-                          m_sig_info->si_addr );
+                          (uintptr_t) m_sig_info->si_addr );
             break;
         case BUS_OBJERR:
             report_error( execution_exception::system_fatal_error,
                           "memory access violation at address: 0x%08lx: object specific hardware error",
-                          m_sig_info->si_addr );
+                          (uintptr_t) m_sig_info->si_addr );
```

Comment:
`uintptr_t` might be unavailable on some compilers

Reply:
You're already using `uintptr_t` in this file: 
https://github.com/boostorg/test/blob/4f3806d80252081bc87e71d97d5b0c8dfd063e17/include/boost/test/impl/execution_monitor.ipp#L1131
https://github.com/boostorg/test/blob/4f3806d80252081bc87e71d97d5b0c8dfd063e17/include/boost/test/impl/execution_monitor.ipp#L87
https://github.com/boostorg/test/blob/4f3806d80252081bc87e71d97d5b0c8dfd063e17/include/boost/test/impl/execution_monitor.ipp#L91